src/js/components/award/AgencyInfo.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * AgencyInfo.jsx
+ * Created by Emily Gullo 01/20/2017
+ **/
+
+import React from 'react';
+import InfoSnippet from './InfoSnippet';
+
+const propTypes = {
+    selectedAward: React.PropTypes.object
+};
+
+export default class AgencyInfo extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            agencyType: "awarding"
+        };
+
+        this.toggleAgency = this.toggleAgency.bind(this);
+    }
+
+    toggleAgency(e) {
+        this.setState({
```

Comment:
When the page URL changes from one award to another, the toggle stays on the value it was at before (in other words, if I was looking at Funding Agency, and then I change the URL to a different award, that award will load with Funding Agency pre-selected instead of the default Awarding Agency). Consider monitoring for award ID changes and reset the `agencyType` state back to `awarding` to handle this case.